### ISSUE TITLE
Firebase Auth Google Provider not Working

### ISSUE THREAD

**Issue by Mikebarson** [firebase/polymerfire#271]
### Description
I'm getting this error whenever I try I to use the this.$.auth.signInWithPopup():
(https://user-images.githubusercontent.com/13252897/30729119-b349156c-9f8f-11e7-9032-c34e3fb39832.png)

### Expected outcome
Should show the sign in page that shows all your google accounts.

### Actual outcome
This error occurs about Network Error
(https://user-images.githubusercontent.com/13252897/30729119-b349156c-9f8f-11e7-9032-c34e3fb39832.png)

**Comment by tjmonsi** [comment]
Hi @Mikebarson  do you have a basic working repo?

**Comment by Mikebarson** [comment]
@tjmonsi I don't have a repo. Do you need the code?

**Comment by tjmonsi** [comment]
I hope so. Are you just using it in localhost or are you calling it  from somewhere (like 127.0.0.1). If the latter, try using localhost

**Comment by FluorescentHallucinogen** [comment]
@Mikebarson Do you use `polymer serve` or `polymer build`?

**Comment by Mikebarson** [comment]
@FluorescentHallucinogen No. I'm directly serving my app in a Django application via views.
